UI/login.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Page Title</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" media="screen" href="css/style.css" />
+    <script src="main.js"></script>
+</head>
+<body>
+    <div class="container">
+        <div class="top-nav">
+            <h2><b>Welcome to Store Manager</b></h2>
+        </div>
+    <div class="login-form">
+    <form action="">
+        <label for="username"><b>Username:</b></label>
+        <input type="text"  placeholder="Enter username.." name="username" required>
+  
+        <label for="password"><b>Password:</b></label>
+        <input type="password" placeholder="Enter Password..." name="username" required>
+
+        <button type="submit">Login</button>
+    </form>
+    </div>
+    </div>
+</body>
+</html>
```

Comment:
Its good practice to add a line at the end of your file before you push to git

Reply:
I'll do that.. Thank you

Reply:
> 
> 
> Its good practice to add a line at the end of your file before you push to git

what do you mean by "add a line?"

Reply:
What @SelaDanti mean is to add a new line at the end of  html file